README.md

```diff
@@ -0,0 +1,23 @@
+# OpenVic-Simulation
+Repo of the OpenVic-Simulation Library for [OpenVic](https://github.com/OpenVicProject/OpenVic)
+
+## Quickstart Guide
+For detailed instructions, view the OpenVic Contributor Quickstart Guide [here](https://github.com/OpenVicProject/OpenVic/blob/master/docs/contribution-quickstart-guide.md)
+
+## Required
+* [scons](https://scons.org/)
+
+## Build Instructions
+1. Install [scons](https://scons.org/) for your system.
+2. Run the command `git submodule update --init --recursive` to retrieve all related submodules.
+3. Run `scons build_ovsim_library=yes` in the project root, you should see a libopenvic-dataloader file in `bin`.
```

Comment:
I think it would be more useful to remove the `build_ovsim_library=yes` argument and direct users to the resulting headless executable. Building the library without headless mode or a larger parent scons project is a very niche use case.